demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1694,7 +1694,6 @@ export class DemoMeetingApp
   async populateAllDeviceLists(): Promise<void> {
     await this.populateAudioInputList();
     await this.populateVideoInputList();
-    await this.populateVideoFilterInputList();
```

Comment:
The `populateAllDeviceLists()`  is also called in the `initializeMeetingSession` method [here[(https://github.com/aws/amazon-chime-sdk-js/blob/c13b6cce5e2d35841b7cb4fac2df2f184a4d45b2/demos/browser/app/meetingV2/meetingV2.ts#L1260) in the demo. So, should we keep it and remove `await this.populateVideoFilterInputList();` from the `onVoiceFocusSettingChanged()`?

Reply:
The change isn't in `onVoiceFocusSettingChanged`, it's in the `authenticate` handler that's configured in `initEventListeners`; GitHub is showing context sub-optimally.

With this change we init VF, the device lists, and the video filter after you authenticate. When you authenticate we _also_ init the device lists. If anything, it's that second `populateAllDeviceLists` call that can go.

Reply:
I removed the additional call in `initializeMeetingSession` and tested.

Reply:
Got it, Github did not show the change under `initEventListeners()`. Thanks!